packages/firestore/CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
 # Unreleased
+- [fixed] Reverted the use of the "crypto" global since it caused some user
```

Comment:
How about: Temporarily reverted the use of `window.crypto` to generate document IDs to address compatibility issues with IE 11 and WebWorkers.

Reply:
Done.